packages/webcert/src/feature/careProvider/CareProviderModal.test.tsx

```diff
@@ -63,13 +63,13 @@ describe('Care provider modal', () => {
     expect(text).not.toBeInTheDocument()
   })
 
-  describe('Tests with common setup', () => {
+  describe('Tests with no logged in unit', () => {
     beforeEach(() => {
       testStore.dispatch(updateUser(getUserWithEmptyUnit()))
       testStore.dispatch(updateUserStatistics(getUserStatistics()))
     })
 
-    it('should show care provider modal if choose unit resource link exists', () => {
+    it('should show care provider modal if choose unit resource link exists and logged in unit is not set', () => {
```

Comment:
Shouldn't the logic check "if logged in unit is not set" belong in the backend code?